db/seeds.rb

```diff
@@ -1,5 +1,9 @@
-
 # Setup user/dev for example site
+developer = FactoryGirl.build :developer
```

Comment:
Seeds are curently broken.
I found that we need to have default developer before creating users and that developer should be found by api key.